docs/UserGuide.md

```diff
@@ -183,30 +222,84 @@ Examples:
-| **Delete** | **Deleting contacts** <br> Format: `del <INDEX NO>` <br> Example: `del 88` <br> <br> **Deleting fields** <br> Format: `del <ANY NUMBER AND TYPE OF FIELDS IN ANY ORDER>` <br> Example: `del p/ personal e/` |
-| **Clear**  | Format: `clear`                                                                                                                                                                                             |
-| **Find**   | Format: `find <FIELD> <VALUE>` <br> Example: `find n/ Michael Scott`                                                                                                                                        |
-| **View**   | Format: `view <INDEX>` <br> Example: `view 5`                                                                                                                                                               |
-| **Back**   | Format: `back`                                                                                                                                                                                              |
-| **List**   | Format: `list`                                                                                                                                                                                              |
-| **Help**   | Format: `help`                                                                                                                                                                                              |
+| **Action**         | **Format, Example**                                                                                                                                                                                         |
+|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**            | Format: `add n/<NAME>` <br> Example: `add n/Alex Dunphy`                                                                                                                                                    |
+| **Edit**           | Format: `edit <ANY NUMBER AND TYPE OF FIELDS IN ANY ORDER>` <br> Example: `edit n/Michael J Wolf pr/her j/Farmer c/FarmingInc ph/999 l/Police t/friend ph/123456`                                           |                                                                                                                                                                                        |
+| **Delete**         | **Deleting contacts** <br> Format: `del <INDEX NO>` <br> Example: `del 88` <br> <br> **Deleting fields** <br> Format: `del <ANY NUMBER AND TYPE OF FIELDS IN ANY ORDER>` <br> Example: `del p/ personal e/` |
```

Comment:
In the same way that we've specified "Create Meeting", "Update Meeting", etc., maybe we could do the same with "Add Contact", "Edit Contact"...